src/applications/hca/selectors.js

```diff
@@ -9,16 +9,31 @@ import { HCA_ENROLLMENT_STATUSES } from './constants';
 // top-level selectors
 export const selectEnrollmentStatus = state => state.hcaEnrollmentStatus;
 export const isEnrollmentStatusLoading = state =>
-  selectEnrollmentStatus(state).isLoading;
+  selectEnrollmentStatus(state).isLoadingApplicationStatus;
+export const isLoadingDismissedNotification = state =>
+  selectEnrollmentStatus(state).isLoadingDismissedNotification;
 export const hasServerError = state =>
   selectEnrollmentStatus(state).hasServerError;
 export const noESRRecordFound = state =>
   selectEnrollmentStatus(state).noESRRecordFound;
 export const isShowingHCAReapplyContent = state =>
   selectEnrollmentStatus(state).showHCAReapplyContent;
+export const isInESR = state => {
+  const status = selectEnrollmentStatus(state).enrollmentStatus;
+  return (
+    [
+      null,
+      HCA_ENROLLMENT_STATUSES.noneOfTheAbove,
+      HCA_ENROLLMENT_STATUSES.canceledDeclined,
+      HCA_ENROLLMENT_STATUSES.deceased,
+    ].includes(status) === false
```

Comment:
I typically suggest people use a `Set` for this kind of check. You might not be able to put null in it, but a Set fits this problem better than an array.

Reply:
okay, will try that.

Reply:
I'm curious why you like using a Set for this instead of an Array. It doesn't seem to be more concise or any more readable.

Reply:
It's not any more concise, but I do think it's more readable, because a Set communicates more information about how you're thinking about these statuses. Namely that: order isn't important, this is a unique set of statuses, and the operations that would be useful on this data are part of the Set api.

We're used to using arrays for this because it was the most concise way to do it pre-ES5 and we're experienced developers. But if you're coming at this problem without that experience, a fair question you might ask when reading this code is, why is this an array? What characteristics of an array made this a better choice than an object, a Map, or a Set? And that's tricky for the reader to figure out, because the answer isn't something you can infer from an understanding of those types.

Reply:
That all totally makes sense, Jeff. Thanks!